docs/tutorials/generate-custom-test-data.rst

```diff
@@ -0,0 +1,120 @@
+.. _ksql_generate-custom-test-data:
+
+Generate custom test data by using the ksql-datagen tool
+========================================================
+
+KSQL enables generating test data that complies with a custom schema

```

Comment:
suggest: `You can use the ksql-datagen tool to generate test data that complies with a custom schema that you define.`